.cargo/config.toml

```diff
@@ -1,2 +0,0 @@
-[net]
-git-fetch-with-cli = true
```

Comment:
If we one day want to use private dependencies. 
e.g. some critical fixes, we will need to install this value again

Reply:
To fix a critical issue we most likely just create a private copy of the repo. We can set the value again if the need arises anyway.

Reply:
I just don't want to fight with it in a rush.